hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/TestHiveSyncTool.java

```diff
@@ -122,117 +116,6 @@ public void teardown() throws Exception {
     HiveTestUtil.clear();
   }
 
-  /**
-   * Testing converting array types to Hive field declaration strings.
-   * <p>
-   * Refer to the Parquet-113 spec: https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#lists
-   */
-  @Test
-  public void testSchemaConvertArray() throws IOException {
```

Comment:
this and 2 tests below already moved to org.apache.hudi.hive.util.TestHiveSchemaUtil. These were added back due to auto merge during rebase probably.